include/grpc++/server_context.h

```diff
@@ -116,6 +117,8 @@ class ServerContext {
 
   std::shared_ptr<const AuthContext> auth_context() const;
 
+  const census_context* get_census_context() const;
```

Comment:
That results in errors like: 

In file included from include/grpc++/stream.h:40:0,
                 from include/grpc++/generic_stub.h:38,
                 from src/cpp/client/generic_stub.cc:34:
include/grpc++/server_context.h:120:42: error: declaration of ‘const census_context* grpc::ServerContext::census_context() const’ [-fpermissive]
   const census_context* census_context() const;
                                          ^
In file included from include/grpc++/impl/call.h:38:0,
                 from include/grpc++/channel_interface.h:40,
                 from include/grpc++/stream.h:37,
                 from include/grpc++/generic_stub.h:38,
                 from src/cpp/client/generic_stub.cc:34:
include/grpc++/client_context.h:51:8: error: changes meaning of ‘census_context’ from ‘struct census_context’ [-fpermissive]
 struct census_context;
        ^
make: *** [objs/opt/src/cpp/client/generic_stub.o] Error 1